docker-compose.yml

```diff
@@ -72,12 +73,20 @@ services:
     labels:
       - "traefik.enable=true"
       - "traefik.docker.network=frontend"
-      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}Mailhog.rule=Host(`mailhog-${COMPOSE_DOMAIN}`)"
-      - "traefik.http.services.${COMPOSE_PROJECT_NAME}Mailhog.loadbalancer.server.port=8025"
+      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}mail.rule=Host(`mail-${COMPOSE_DOMAIN}`)"
+      - "traefik.http.services.${COMPOSE_PROJECT_NAME}mail.loadbalancer.server.port=8025"
 
   redis:
```

Comment:
Should we move redis and node to docker-compose.override.yml ?

Reply:
That would be the ITK Dev way to do it, but https://github.com/os2display/display-api-service/commit/04cbc36771a7b7020b148ffa035876223571c79d